resources/js/utils/score-helper.ts

```diff
@@ -105,6 +105,67 @@ export function rank(score: SoloScoreJson) {
     : score.rank;
 }
 
+export function rankCutoffs(score: SoloScoreJson): number[] {
+  // <rank>: minimum acc => (higher rank acc - current acc)
+  // for SS, use minimum accuracy of 0.99 (any less and it's too small)
+  // actual array is reversed as it's rendered from D to SS clockwise
+
+  if (shouldReturnLegacyValue(score)) {
+    return {
```

Comment:
just a matter of style but I wonder if a switch is easier to read. And maybe a different function to calculate the result array from the actual accuracy requirements - transforming the 0.99, 0.9801, etc to the expected values - while at it.

Reply:
I made an attempt in https://github.com/ppy/osu-web/pull/11536/commits/a86ce4138c86fa0a834860f180321a0587703228, I guess see if you like it better?

Reply:
yeah that's definitely easier to understand 😄 

(the switch still kinda sucks but that's just javascript)